rs.json

```diff
@@ -1,206 +1,206 @@
 {
-  "languageCode": "sr",
-  "languageName": "Serbian",
-  "langageNameShort": "SRB",
-  "dateLocale": "enUS",
+  "languageCode": "sr_latn",
+  "languageName": "Serbian (Latin)",
+  "langageNameShort": "SRB_LATN",
+  "dateLocale": "sr",
```

Comment:
Ali can you confirm this exists? Not sure on what list to check.

Reply:
Let's make it:
```
  "languageCode": "sr",
  "languageName": "Serbian (Latin)",
  "langageNameShort": "SRB_LATN",
  "dateLocale": "sr-Latn"
```